### ISSUE TITLE
mAP=0 in step2

### ISSUE THREAD

**Issue by Jilliansea** [xuannianz/EfficientDet#52]
Thanks for your code first. I have some problems when train with PASCAL VOC.
In the first step I can train the network smoothly and the best mAP= 0.546 in epoch 47;
then I start step 2, train with h5 of epoch47, but the mAP is always 0.000 in the first 3 epoch, then  I have to stop the training. So do you know what happend? Thanks very much
step1:
python3 train.py --snapshot imagenet --phi 1 --gpu 0 --random-transform --compute-val-loss --freeze-backbone --batch-size 2  --steps 1000 pascal data/VOC2012
step2:
python3 train.py --snapshot checkpoints/2019-12-25/pascal_47_1.4936_1.5473.h5 --phi 1 --gpu 0 --random-transform --compute-val-loss --freeze-bn --batch-size 4 --steps 10000 pascal data/VOC2012

**Comment by Jilliansea** [comment]
I solved the problem. It's because I haven't change the learning rate 1e-3 --> 1e-4;
after I change the learning rate, the mAP of the epoch1 of stage2 turns to 0.5993
Thx
